components/common/listingCart/ListingCartModal.vue

```diff
@@ -0,0 +1,196 @@
+            no-shadow
+            class="is-flex is-flex-grow-1 btn-height"
+            @click.native="confirm" />
+        </div>
+      </div>
+    </NeoModal>
+  </div>
+</template>
+<script setup lang="ts">
+import { Interaction } from '@kodadot1/minimark/v1'
+import ListingCartPriceInput from '@/components/common/listingCart/ListingCartPriceInput.vue'
+import { totalPriceUsd } from '@/components/common/shoppingCart/utils'
+import IdentityItem from '@/components/identity/IdentityItem.vue'
+import CommonTokenMoney from '@/components/shared/CommonTokenMoney.vue'
+import { NeoButton, NeoModal } from '@/libs/ui'
+import { usePreferencesStore } from '@/stores/preferences'
+import { TokenToList } from '@/composables/transaction/types'
+import { useListingCartStore } from '@/stores/listingCart'
+import { calculateBalance } from '@/utils/format/balance'
+import { sum } from '@/utils/math'
+import { warningMessage } from '@/utils/notification'
+const { isLogIn, accountId } = useAuth()
+const { urlPrefix } = usePrefix()
+const preferencesStore = usePreferencesStore()
+const listingCartStore = useListingCartStore()
+const { transaction, isLoading, status } = useTransaction()
+const { $i18n } = useNuxtApp()
+
+const fixedPrice = ref()
+function setFixedPrice() {
+  listingCartStore.setFixedPrice(fixedPrice.value)
+  fixedPrice.value = undefined
+}
+
+const priceUSD = computed(() => {
+  const { nfts, royalties } = totalPriceUsd(listingCartStore.itemsInChain)
+  return (nfts + royalties).toFixed(2)
+})
+
+const totalNFTsPrice = computed(() =>
+  sum(listingCartStore.itemsInChain.map((nft) => Number(nft.price)))
+)
+
+const confirmListingLabel = computed(() => {
+  switch (listingCartStore.incompleteListPrices) {
+    case 0:
+      return 'Complete Listing'
+    case 1:
+      return '1 Item Is Missing Price'
+    default:
+      return `${listingCartStore.incompleteListPrices} Items Are Missing Price`
+  }
+})
+async function confirm() {
+  const token = [] as TokenToList[]
+  for (const item of listingCartStore.itemsInChain) {
+    if (item.listPrice) {
+      token.push({
+        price: String(calculateBalance(item.listPrice)),
+        nftId: item.id,
+      })
+    }
+  }
+  try {
+    await transaction({
+      interaction: Interaction.LIST,
+      urlPrefix: urlPrefix.value,
+      token,
+      successMessage: $i18n.t('transaction.price.success') as string,
+      errorMessage: $i18n.t('transaction.price.error') as string,
+    })
+    listingCartStore.clear()
+    preferencesStore.listingCartModalOpen = false
+  } catch (error) {
+    warningMessage(error)
+  }
+}
+</script>
+<style lang="scss" scoped>
+@import '@/styles/abstracts/variables';
+
+.height-100 {
+  height: 100%;
+}
+
+.top {
+  z-index: 1000;
+}
+
+.shade-border-color {
+  @include ktheme() {
+    border-color: theme('k-shade');
+  }
+}
+
+.rounded {
+  border-radius: 10rem;
+}
+
+.modal-width {
+  width: 25rem;
+}
+.btn-height {
+  height: 3.5rem;
+}
```

Comment:
:/ fixed size

Reply:
i'll change btn height to padding and width to max-width. Would that be okay?